src/test/java/com/coherentsolutions/aqa/java/api/makarevich/ZipCodeTest.java

```diff
@@ -0,0 +1,98 @@
+package com.coherentsolutions.aqa.java.api.makarevich;
+
+import com.coherentsolutions.aqa.java.api.makarevich.httpClient.HttpResponseWrapper;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.util.ArrayList;
+
+import static com.coherentsolutions.aqa.java.api.makarevich.configuration.Configuration.API_ZIPCODES_ENDPOINT;
+import static com.coherentsolutions.aqa.java.api.makarevich.configuration.Configuration.API_ZIPCODES_EXPAND_ENDPOINT;
+import static com.coherentsolutions.aqa.java.api.makarevich.constants.Constants.STATUS_CODE_200_OK;
+import static com.coherentsolutions.aqa.java.api.makarevich.constants.Constants.STATUS_CODE_201_CREATED;
+import static java.lang.System.currentTimeMillis;
+import static org.junit.jupiter.api.Assertions.*;
+
+public class ZipCodeTest extends TestBase {
+    /*
+    Scenario #1
+    Given I am authorized user
+    When I send GET request to /zip-codes endpoint
+    Then I get 200 response code
+    And I get all available zip codes in the application for now
+    */
+
+    @Test
+    public void testGetZipCodes() throws IOException {
+        HttpResponseWrapper response = httpMethods.get(API_ZIPCODES_ENDPOINT);
+        ArrayList<String> arrayList = response.getReponseBodyAsArray();
```

Comment:
I feel like you need one more level. Class where you will send request to specific endpoint and get response. 
Something like ZipCodeClient. 
It will allow to write high level code on the test level without knowing details about URL and type or response. 
The method will be something like zipCodeClient.getZipCodes().
You can move add status code as method parameter and do status code validation there. In such case you will need validate it with throwing expection. 
Or you can return response and status code on the test level and do validation as you are doing now

Reply:
Introduced one more level to make it more high level in tests classes. Moved all logic to ZipCodeService class. Let me know if it does not work and you mean different thing.